### ISSUE TITLE
ABNORMAL status when solving simple model with CLP using Barrier algorithm 

### ISSUE THREAD

**Issue by fadulalla** [google/or-tools#2479]
**What version of OR-Tools and what language are you using?**
Version: 8.2.8710
Language: C#

**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**
CLP (`OptimizationProblemType.CLP_LINEAR_PROGRAMMING`)

**What operating system (Linux, Windows, ...) and version?**
Microsoft Windows 10 Enterprise 2016 LTSB
Version: 10.0.14393 Build 14393

**What did you do?**
Attempted to solve the below LP model using the CLP linear solver using the barrier method (this also happens with default parameters).
```
* Generated by MPModelProtoExporter
*   Name             : Test Solver
*   Format           : Free
*   Constraints      : 0
*   Variables        : 5
*     Binary         : 0
*     Integer        : 0
*     Continuous     : 5
NAME          Test Solver
OBJSENSE
  MAX
ROWS
 N  COST
COLUMNS
    V0    COST       650
    V1    COST        70
    V2    COST      1670
    V3    COST       830
    V4    COST      1300
BOUNDS
 LO BOUND   V0       0.4
 UP BOUND   V0         7
 LO BOUND   V1       0.4
 UP BOUND   V1         7
 LO BOUND   V2       0.4
 UP BOUND   V2         7
 LO BOUND   V3       0.4
 UP BOUND   V3         7
 LO BOUND   V4       0.4
 UP BOUND   V4         7
ENDATA
```
using the following code:
```cs
using (var solver = new Solver("Test Solver", OptimizationProblemType.CLP_LINEAR_PROGRAMMING)) {
   solver.EnableOutput();
   Objective objective = solver.Objective();
   objective.SetMaximization();
   ... //set variables and constraints
   MPSolverParameters parameters = new MPSolverParameters();
   parameters.SetIntegerParam(MPSolverParameters.IntegerParam.LP_ALGORITHM, 12 /*Barrier*/); 
   var resultStatus = solver.Solve(parameters);
}
```

**What did you expect to see**
`OPTIMAL` with the objective function having the value of `31640` and all 5 variables having the value of `7.0`.

**What did you see instead?**
`ABNORMAL`, with `solve.EnableOutput()` not printing anything to the console as to why it is abnormal.

**Anything else we should know about your project / environment**
I have tried using the default parameters (I assume it's simplex by default?) to no avail, still ABNORMAL.
Also, I have tried using GLOP solver and it solves optimally with the exact expected results.
I have tested the model using a different solver (using the barrier algorithm) and it's optimal with the expected results.

I need the Barrier algorithm for a larger model and thought I'd test out a smaller model first, so I was surprised to see abnormal on such a trivial model. What am I doing wrong?
(Also I'm not sure I need to submit a CLA considering I'm asking for help rather than contributing? Please let me know and I'll be happy to do so!)

**Comment by lperron** [comment]
Send the model to the clp team.

**Comment by fadulalla** [comment]
Thanks, will do! Though it would have been nice if ortools provided more information other than just abnormal
Have a good weekend!

**Comment by fadulalla** [comment]
I've investigated a little bit, and here are my findings in case someone comes across this issue in the future:
It returns abnormal when you have no constraints, adding a `solver.MakeConstraint("dummy");` made it solve optimally.
Using clp outside of ortools, the above mps model will not solve either as clp will complain about a `Bad image on line 20 < BOUNDS >`. I had to add an empty `RHS` section for it to solve.
